src/api/firebase.js

```diff
@@ -124,9 +126,11 @@ export async function createList(userId, userEmail, listName) {
 
 	const userDocumentRef = doc(db, 'users', userEmail);
 
-	updateDoc(userDocumentRef, {
+	await updateDoc(userDocumentRef, {
 		sharedLists: arrayUnion(listDocRef),
 	});
+
+	return listDocRef.path;
```

Comment:
Great idea to return useful data from this function! 

One tiny thought is to return the entire `listDocRef` and then let the consumer parse out what they need (in this case `listDocRef.path`) just to keep the interface of your function a little more flexible.  It makes logical sense to me for a function called `createList` to return a reference to the list that was created. Not a change that needs to happen, but just something to think about when you are writing functions.